### PR TITLE
Custom 404 page implementation

### LINKED ISSUE #99: [SEO]  404 page improve the SEO prespective
Custom "404 page" improve the user experience  as well as the SEO .
Currently it is default nextjs 404 page 
image

### PR DESCRIPTION
**Pull request by AQIB-NAWAB**
**What kind of change does this PR introduce?**
Developed a 404 page for better user experience 

**Issue Number:**
closes #99

**Screenshots/videos:**
Here is output of new custom 404 page 
image

http://localhost:3000/unknown-page

**Summary**

My main motivation to add 404 page was improving the better user experience

### CODE REVIEW

**Comment by DhairyaMajmudar** [comment]
Thank you @AQIB-NAWAB for thoughtful PR can you pls. keep the designs matching the existing json schema color themes and componets

**Comment by AQIB-NAWAB** [comment]
fix #99

**Review by JeelRajodiya — CHANGES_REQUESTED** [review]
Can you update the following
- Make sure the font size is as large as we have in the design 
- Fix alignment of the subtitle as per the design 
image
- make the borders rounded in the back to home button (as per the design)

File: app/not-found.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import styles from './styles/404.module.css';
+import Link from 'next/link';
+import { useEffect } from 'react';
+
+export default function Custom404() {
+    const { colorMode } = useColorMode();
+    
+    useEffect(() => {
+        document.documentElement.setAttribute('data-theme', colorMode);
+    }, [colorMode]);
```
[the remark this answers is not in the document]
Comment (AQIB-NAWAB) [reply app/not-found.tsx]: No actually we don't need this one

**Comment by AQIB-NAWAB** [comment]
Here is update with figma values 
image
image

**Comment by JeelRajodiya** [comment]
Looks good to me!

**Pull request merged**